src/cli/onefuzz/templates/__init__.py

```diff
@@ -7,9 +7,10 @@
 import os
 import tempfile
 import zipfile
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
With the removal of the code now used by `JobMonitor`, this is not needed.

```suggestion
from typing import Any, Dict, List, Optional
```